wpimath/src/test/java/edu/wpi/first/math/trajectory/TrajectoryGeneratorTest.java

```diff
@@ -86,11 +82,11 @@ void testQuinticCurvatureOptimization() {
     Trajectory t =
         TrajectoryGenerator.generateTrajectory(
             List.of(
-                new Pose2d(1, 0, Rotation2d.fromDegrees(90)),
-                new Pose2d(0, 1, Rotation2d.fromDegrees(180)),
+                new Pose2d(1, 0, Rotation2d.kCCW_Pi_2),
+                new Pose2d(0, 1, Rotation2d.kPi),
                 new Pose2d(-1, 0, Rotation2d.fromDegrees(270)),
-                new Pose2d(0, -1, Rotation2d.fromDegrees(360)),
```

Comment:
Is 360 == 0 in Rotation2d? I remember that it was debated some time ago.

Reply:
It's supposed to be, though we inherited some tech debt from 254 with the `value` field that means the Euler angle getter retains the original value instead of being recalculated from the cos/sin unit vector.

Reply:
So is replacing 360° with kZero correct or not?

Reply:
It's case-specific. If the code in question never calls an Euler angle getter, then replacing 360 with 0 is fine.